server/serializer/channel_subscription.go

```diff
@@ -134,23 +163,62 @@ func SubscriptionsFromJSON(bytes []byte) (*Subscriptions, error) {
 	return subs, nil
 }
 
+func OldSubscriptionsFromJSON(bytes []byte) (*OldSubscriptions, error) {
+	var subs *OldSubscriptions
+	if len(bytes) != 0 {
+		if unmarshalErr := json.Unmarshal(bytes, &subs); unmarshalErr != nil {
+			return nil, unmarshalErr
+		}
+	} else {
+		subs = NewOldSubscriptions()
+	}
+	return subs, nil
+}
+
 func FormattedSubscriptionList(channelSubscriptions StringSubscription) string {
```

Comment:
I think it might be easy having some tests for this, and it saves a lot of time to understand what's the output format, as the test is almost self-documenting the methods output